collections/collections.go

```diff
@@ -25,7 +25,7 @@ var (
 	dsAppTokensKey = datastore.NewKey("/apptokens")
 	dsAppUsersKey  = datastore.NewKey("/appusers")
 
-	dsFilesKey = datastore.NewKey("/files")
+	dsFoldersKey = datastore.NewKey("/folders")
```

Comment:
We only need to track folders in the thread. The files are available from the IPFS API. This also helps decouple Threads from the underlying Pinning API... we only have to track to roots, not each file.